web/components/receiptForm.tsx

```diff
@@ -31,11 +32,69 @@ export const ReceiptForm = () => {
           </DrawerHeader>
           <div className="p-4">
             <form className="flex flex-col gap-4">
-              <Input required type="number" label="Subtotal" startContent="$" />
-              <Input required type="number" label="GST/HST" startContent="$" />
-              <Input type="number" label="Gratuity" startContent="$" />
-              <Input required type="date" label="Date" />
-              <Input disabled type="number" label="Total" />
+              <div className="flex items-center justify-between">
+                <div className="flex items-center">
+                  <label className="text-sm">Subtotal</label>
+                  <Tooltip content="The pre-tax ammount">
+                    <CircleHelp className="w-4 h-4 ml-2" />
+                  </Tooltip>
+                </div>
+                <Input
+                  required
+                  type="number"
+                  startContent="$"
+                  placeholder="0.00"
+                  className="max-w-52"
+                />
+              </div>
+              <div className="flex items-center justify-between">
+                <div className="flex items-center">
+                  <label className="text-sm">GST/HST</label>
+                  <Tooltip content="Goods and services tax/harmonized sales tax">
+                    <CircleHelp className="w-4 h-4 ml-2" />
+                  </Tooltip>
+                </div>
+                <Input
+                  required
+                  type="number"
+                  startContent="$"
+                  placeholder="0.00"
+                  className="max-w-52"
+                />
+              </div>
+              <div className="flex items-center justify-between">
+                <div className="flex items-center">
+                  <label className="text-sm">Gratuity</label>
+                  <Tooltip content="The gratuity provided on this expense">
+                    <CircleHelp className="w-4 h-4 ml-2" />
+                  </Tooltip>
+                </div>
+                <Input
+                  type="number"
+                  startContent="$"
+                  placeholder="0.00"
+                  className="max-w-52"
+                />
+              </div>
+              <div className="flex items-center justify-between">
+                <label className="text-sm">Date</label>
+                <Input required type="date" className="max-w-52" />
+              </div>
+              <div className="flex items-center justify-between">
+                <div className="flex items-center">
+                  <label className="text-sm">Total</label>
+                  <Tooltip content="The sum of all items in this expense">
+                    <CircleHelp className="w-4 h-4 ml-2" />
+                  </Tooltip>
+                </div>
+                <Input
```

Comment:
Should we indicate to the user that the total is auto-calculated and can't be edited? Either changing the shade of the box or something along those lines?

Reply:
This field is already disabled and users cant input anything to it. See `isReadOnly` on line 91.

Reply:
I meant more like a visual indicator so the user doesn't think it should be editable

Reply:
Maybe including something in the tooltip would be enough? Or do you think a change to the actual textbox would be better.